.editorconfig

```diff
@@ -0,0 +1,24 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
Did you mean to commit this file?

Reply:
An editorconfig file is just good practice for any shared development.  However, this can be removed. The file works in conjunction with most IDE solutions to automatically format files and keep consistency across multiple development environments and platforms.

Reply:
Keep it in place if it's deliberate.